Tests/TensorFlowTests/OperatorTests/BasicTests.swift

```diff
@@ -287,26 +288,27 @@ final class BasicOperatorTests: XCTestCase {
         let array1D = slice1D.array
 
         /// Test shapes
-        XCTAssertEqual([1, 4, 5], array3D.shape)
-        XCTAssertEqual([2, 5], array2D.shape)
-        XCTAssertEqual([2], array1D.shape)
+        XCTAssertEqual(array3D.shape, [1, 4, 5])
+        XCTAssertEqual(array2D.shape, [2, 5])
+        XCTAssertEqual(array1D.shape, [2])
 
         /// Test scalars
         XCTAssertEqual(
```

Comment:
@rxwei @eaplatanios this seems to be throwing the build error, should I wrap the addition in a parentheses of its own?